api/src/main/resources/routes

```diff
@@ -3,6 +3,8 @@ GET /ping dcos.metronome.api.v1.controllers.ApplicationController.ping
 GET /info dcos.metronome.api.v1.controllers.ApplicationController.info
 GET /v1/metrics dcos.metronome.api.v1.controllers.ApplicationController.showMetrics
 
+GET /v1/deployments dcos.metronome.api.v1.controllers.QueueController.queue
```

Comment:
switch this to `/v1/queue`

Reply:
good point, because for jobs there is no notion of "deployment" anyway...